ros_cross_compile/ros_cross_compile.py

```diff
@@ -125,6 +124,14 @@ def parse_args(args: List[str]) -> argparse.Namespace:
              'when running repeatedly during development, but has undefined behavior if'
              "the workspace's dependencies have changed since the last time"
              'collection was run.')
+    parser.add_argument(
+        '--skip-rosdep-keys',
+        required=False,
+        default=[],
+        type=str,
```

Comment:
this type isn't correct, it outputs a list. you can just skip specifying this argument